src/eru.WebApp/Resources/Pages/Index.en.resx

```diff
@@ -0,0 +1,65 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<root>
+    <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+        <xsd:element name="root" msdata:IsDataSet="true">
+            
+        </xsd:element>
+    </xsd:schema>
+    <resheader name="resmimetype">
+        <value>text/microsoft-resx</value>
+    </resheader>
+    <resheader name="version">
+        <value>1.3</value>
+    </resheader>
+    <resheader name="reader">
+        <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <resheader name="writer">
+        <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <data name="who-are-we" xml:space="preserve">
+        <value>Who are we?</value>
+        <comment>who-are-we</comment>
+    </data>
+    <data name="who-are-we-content" xml:space="preserve">
+        <value>Welcome! We, Stanislaw Nieradko (XXLO IIb1) and Blazej Sak (XXLO IIc1), know that the information about substitution reaches us, students, too slowly. We decided to do something about it. For this reason we created eru. A platform to quickly send information about substitutions to all interested students. We hope that you will like our platform and enjoy using it. Greetings,</value>
+        <comment>who-are-we-content</comment>
+    </data>
+    <data name="eru-team" xml:space="preserve">
+        <value>eru developement team</value>
+        <comment>eru-team</comment>
+    </data>
+    <data name="how-to-get-started" xml:space="preserve">
+        <value>How to get started?</value>
+        <comment>how-to-get-started</comment>
+    </data>
+    <data name="get-started-1" xml:space="preserve">
+        <value>Getting started is very simple! Just select one of available platforms that we support (we can provide support for a variaty of platforms, if the one you're using is missing, reach out to us! We'll gladly help you with that ;)) With every platform we've created a simple intruction how to subscribe to your classes substitutions. Usually you just need to start a conversation with our platform which will enable you to register to our services.</value>
```

Comment:
```suggestion
        <value>Getting started is very simple! Just select one of available platforms that we support (we can provide support for a variety of platforms, if the one you're using is missing, reach out to us! We'll gladly help you with that ;)) With every platform we've created a simple intruction how to subscribe to notifications. Usually you just need to start a conversation with our bot.</value>
```
In my opinion: 
(1) noun 'bot' is more appropriate here (instead of 'platform').
(2) The last sentence should not repeat the previous one.